src/templates/components/button.html

```diff
@@ -3,7 +3,7 @@
     name="{{ componentId }}"
     ng-class="{'btn-block': component.block}"
     class="btn btn-{{ component.theme }} btn-{{ component.size }}"
-    ng-disabled="readOnly || formioForm.submitting || (component.disableOnInvalid && formioForm.$invalid) || hasError()"
+    ng-disabled="readOnly || formioForm.submitting"
```

Comment:
This is going to break behavior where the button is supposed to disable when an error occurs to prevent double submission. I think by taking this we will introduce a worse problem of double submitting errors.